### ISSUE TITLE
Game is save to leave, but it wasn't 

### ISSUE THREAD

**Issue by DipSwitch** [ValveSoftware/Dota2-Gameplay#1773]
### Description

In the chat a message came by that said: The game is save to leave now
When you came to in the menu there wasn't a text: This game is save to leave, but there was a text: Leaving this game can have consequences

Normally when they say "The game is save to leave" the text is green and says you can leave.
This is not the first time this bug has serviced.
The best part is, one of the enemy team didn't leave and won by himself by controlling all other hero's... 

### Example Match ID (and possibly Timestamp)

6657614485 - +/- 02:00

### Screenshots

![image](https://user-images.githubusercontent.com/476484/178375758-fe52b02a-bd4d-4635-a76a-55b55f8b9a55.png)

![image](https://user-images.githubusercontent.com/476484/178375858-83ae97c0-dfc7-464b-9c02-4bd4d3e925cd.png)
Should be "not stored" I suppose

**Comment by DipSwitch** [comment]
I do have to read a bit faster. I'm dyslectic so I never got to the part where they say: Leavers will be scored as losers... :D

**Comment by YaphzOr** [comment]
Like you said, since you are dyslexic you may have misjudged the situation since DotA 2 is working as intended there was no bug nor was anyone trolling you. It's simply saying you can leave this match but the game will count despite players leaving. Not a bug or an issue.

**Comment by DipSwitch** [comment]
Yes it's a bug, because when I switched to the menu to **Disconnect** and **Abandon** it showed below the button that **abandoning could result in consequences.** instead of what it normally shows: **This game is save to leave** or something like that.

**Comment by DipSwitch** [comment]
Probably the same as issue #1779

**Comment by YaphzOr** [comment]
> Yes it's a bug, because when I switched to the menu to **Disconnect** and **Abandon** it showed below the button that **abandoning could result in consequences.** instead of what it normally shows: **This game is save to leave** or something like that.

I understand you. Re-read my comment so you'll understand what I said.

**Comment by DipSwitch** [comment]
No you don't understand, if the game says:

**Comment by AnanasSok** [comment]
It seems that there is a cheat that can send such a message.

**Comment by DipSwitch** [comment]
Magnus really did disconnect because there was a disconnected plug over his image.

But ok, tricky that such a cheat / hack exists. So the text in the menu is always leading. And the message in chat could give a hint you can disconnect.
